src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,21 +2,26 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called. Params: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("AuthenticationException was thrown."
+                    + " Username or password are incorrect");
```

Comment:
I think this is bad practice. Watch the video "Logging best practices" (8 minutes)

Reply:
agree. besides, you are duplicating the message